.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 .idea/
 *.iml
 build/
+
```

Comment:
Why extra line?

Reply:
Accidental, this is why I should probably not try programming at 3am lol. Just undid the change with a new commit